internal/version/vcs_go1.18.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2022 The go-ethereum Authors
+// This file is part of the go-ethereum library.
+//
+// The go-ethereum library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The go-ethereum library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
+
+//go:build go1.18
+// +build go1.18
```

Comment:
Will it be compiled with 1.19? Or it's just only for 1.18

Reply:
The way it's written, I think, we'll have to keep adding future version. So instead of `1.18 vs !1.18` which later becomes ` (1.19 || 1.18) vs !(1.18 || 1.19)`, perhaps we should just list the two that are not supported, now and forever: 
`!(1.16 || 1.17) vs (1.16 || 1.17)`.

Reply:
EDIT: No, that's now how it works. @fjl is correct - because the directive `go1.18` means 1.18 and higher.